extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -481,15 +481,29 @@ export class AmpDatePicker extends AMP.BaseElement {
 
     const min = mutations['min'];
     if (min !== undefined) {
+      this.element.setAttribute('min', min);
```

Comment:
`amp-bind` should automatically do this (https://github.com/ampproject/amphtml/blob/5d09bae540bc116b0ddf1782db0a85d589ad4c3f/src/purifier.js#L589) are you seeing otherwise?

Reply:
I am seeing this. I don't know why I thought it wasn't doing that. Thanks